src/pages/settings/Profile/PersonalDetails/DateOfBirthPage.js

```diff
@@ -67,10 +66,7 @@ class DateOfBirthPage extends Component {
         if (params && params.year) {
             this.setState({selectedYear: params.year});
             if (this.datePicker) {
-                this.datePicker.focus();
-                if (_.isFunction(this.datePicker.click)) {
-                    this.datePicker.click();
-                }
+                this.datePicker.showPicker();
```

Comment:
This introduced a regression https://expensify.slack.com/archives/C049HHMV9SM/p1680625096406979 The reason is that the text input is now no longer focused so no blur event will be emitted and the calendar will not hide on clicking outside. Fix:
```diff
diff --git a/src/pages/settings/Profile/PersonalDetails/DateOfBirthPage.js b/src/pages/settings/Profile/PersonalDetails/DateOfBirthPage.js
index 4983c2cc5f..51d3dc5c55 100644
--- a/src/pages/settings/Profile/PersonalDetails/DateOfBirthPage.js
+++ b/src/pages/settings/Profile/PersonalDetails/DateOfBirthPage.js
@@ -67,6 +67,7 @@ class DateOfBirthPage extends Component {
             this.setState({selectedYear: params.year});
             if (this.datePicker) {
                 this.datePicker.showPicker();
+                this.datePicker.textInputRef.focus();
             }
         }
     }
```

Reply:
Discussing in slack I dont think this is a regression, we have a different issue.